internal/apikeys/functions_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build testing
+
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+package apikeys
+
+import (
+	"encoding/base64"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestHashKeys(t *testing.T) {
```

Comment:
I don't see much point in this test. It essentially tests the `sha256` hashing function that we use as an external dependency. I suggest we remove this test altogether.

Reply:
so is the on focus writing tests for codes created by daytona, i mean codes that serve as internal dependency's

Reply:
Yes but testing functions that are already tested should not be the goal here